### PR TITLE
Delete a redundant arg and optimize some codes in the `pkg/cmd/admin/registry/registry.go`

### PR DESCRIPTION
**Pull request by miaoyq**
1. Delete the arg `namespace` in the `generateSecretsConfig`, because it is not used. 
2. Refactor some codes in the `RunCmdRegistry`, make it more clean.

### CODE REVIEW

**Comment by miaoyq** [comment]
@smarterclayton  Can this PR be merged?  thanks!

File: pkg/cmd/admin/registry/registry.go [hunk 1]
```diff
@@ -300,21 +300,16 @@ func (opts *RegistryOptions) RunCmdRegistry() error {
...
+			return fmt.Errorf("registry does not exist; the provided credentials %q are missing the client certificate and/or key", opts.Config.Credentials)
 		}
 
 		secretEnv = app.Environment{
 			"OPENSHIFT_MASTER":    config.Host,
 			"OPENSHIFT_CA_DATA":   string(config.CAData),
 			"OPENSHIFT_KEY_DATA":  string(config.KeyData),
 			"OPENSHIFT_CERT_DATA": string(config.CertData),
-			"OPENSHIFT_INSECURE":  insecure,
+			"OPENSHIFT_INSECURE":  fmt.Sprintf("%v", config.Insecure),
```
Comment (smarterclayton) [inline pkg/cmd/admin/registry/registry.go]: It would probably be better to use `%t` here because we want Insecure to be flagged as a bool.

Comment (miaoyq) [inline pkg/cmd/admin/registry/registry.go]: Thanks for review!  @smarterclayton 
Updated.

**Comment by smarterclayton** [comment]
LGTM [merge] thanks

**Comment by miaoyq** [comment]
@smarterclayton  This PR isn't merged still,  could you please trigger another test and merge process? Thanks!

**Comment by smarterclayton** [comment]
[test]

**Comment by smarterclayton** [comment]
[merge]

**Pull request merged**